hvantk/commands/hgc_cli.py

```
@@ -391,3 +397,975 @@ def mt2vcf(ctx, input, output, filter_adj, min_ac, split_multi, dry_run):
+            except ValueError as e:
+                error_msg = f"Invalid value or data: {e}"
+                click.echo(f"❌ Error processing {file_stem}: {error_msg}")
+                return {'file': file_stem, 'success': False, 'error': error_msg}
+            except ImportError as e:
+                error_msg = f"Import error: {e}"
+                click.echo(f"❌ Error processing {file_stem}: {error_msg}")
+                return {'file': file_stem, 'success': False, 'error': error_msg}
+            except Exception as e:
+                # Catch any unexpected errors and log them
+                error_msg = f"Unexpected error ({type(e).__name__}): {e}"
+                logger.exception(f"Unexpected error processing {file_stem}: {e}")
```

Ruff (0.14.3)

Redundant exception object included in `logging.exception` call

(TRY401)